package.json

```diff
@@ -63,9 +63,10 @@
     "prettier-config-standard": "^5.0.0",
     "ts-node": "^10.8.1",
     "turbo": "^1.3.1",
-    "typescript": "^4.6.3"
+    "typescript": "^4.7.4"
   },
   "resolutions": {
-    "trim": "0.0.3"
+    "trim": "0.0.3",
+    "@types/react": "^18.0.14"
```

Comment:
[F] This does scare me but without it we cannot upgrade (webview toolkit and react-virtualization have incompatible types). 

Our use of the toolkit still seems to work. I've checked the virtualized plots story in storybook and it seems to render just fine. Drag and drop is a bit janky.